src/app/services/baw-api/api-common.ts

```diff
@@ -71,6 +71,7 @@ export type Filter = "filter";
 export const emptyParam: Empty = "";
 export const newParam: New = "new";
 export const filterParam: Filter = "filter";
+export const commitParam = "?commit";
```

Comment:
this is not a common pattern, this constant should probably not be in common. Move to AEIF service